CONTRIBUTING.md

```diff
@@ -87,184 +61,63 @@ A: There is no strict definition however examples of substantive changes are:
 
 
 Q: How should I prepare for an RFC?
 
-A: Bring the idea to the [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ) channel and talk it over with the core team.
+A: Bring the idea to the [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ) channel or the Scrum of Scrums meeting and talk it over with the core team.
 
 
 Q: What if my RFC is not accepted?
 
-A: It will be closed and can be reopened if it is updated to address the items that prevented it's acceptance.
+A: It will be closed, but can be reopened if it is updated to address the items that prevented it's acceptance.
 
 
 Q: What if my team doesn’t have the resources to implement our accepted RFC? 
 
 A: Anyone can coordinate with you and the core team to take over the work. 
 
-Setting up the FHIR server locally
----
-Requirements: Docker 
-
-Caution: Setting up your local environments requires git patches to run. Please make sure you `make unservable` and `make unloadadable` before you commit your changes to revert these patches. 
-
-Let's begin!
-
-Clone the repository onto you're computer.
-```
-git clone https://github.com/CMSgov/beneficiary-fhir-data
-```
-
-The instructions from here on should be run from the `contributing` directory in repository.
-
-To simply run tests or execute other tasks in the BFD bring up the docker containers.
-Note: As a prerequisite, the bfd Docker environments need a few variables to be set in a file named .env placed within the /contributing directory.
-
-- `BFD_DIR` specifies the directory on your host machine where you have cloned https://github.com/CMSgov/beneficiary-fhir-data
-- (optional) `SYNTHETIC_DATA` specifies a folder where you have the full set of synthetic rif files.
-- (defaults to `/app`) `BFD_MOUNT_POINT` the path within the service container where the beneficiary-fhir-data directory will be mounted.
-
-Here's an example `.env` file that docker-compose could use:
-```
-BFD_DIR=../../beneficiary-fhir-data
-SYNTHETIC_DATA=../../synthetic-data
-```
-
-```
-make up
-```
-
-This brings services up in the background and displays the logs from the `bfd` container. Once the logs show that that the system is started (this can take a minute or so depending on your machine) the logs can be exited with Ctrl+C.
-
-Now the system can be interacted with. Here's an example of running tests for the `bfd-server` module.
-
-```
-docker-compose exec bfd bash
-cd /app/apps/bfd-server
-mvn verify
-```
-
-### Serving the BFD
-
-To run the BFD locally in a way that will allow you and other systems to interact with it some modifications need to be made so that it serves on a consistent port. Caution: Since this changes the code in the repository (server-start.sh) please keep in mind not to commit these changes.
-
-These changes are contained in the file `contributing/patches/allow_local_port_config.patch` and can be applied with 
-
-```
-make servable
-```
-To undo the changes run `make unservable`.
+### Getting started
```

Comment:
I'd suggest replacing this entire section with a brief pointer to one of these:

* <https://github.com/CMSgov/beneficiary-fhir-data/blob/master/README.md>
* <https://github.com/CMSgov/beneficiary-fhir-data/blob/master/dev/devenv-readme.md>

Reply:
@cbrunefearless @keithdadkins @jzulim any preference?

Reply:
The norm is to have a "Getting started" section in the main README that points to the CONTRIBUTING docs, or if it's simple enough, having the entire "getting started" instructions in the README (this is not likely what we want to do). 

If we want to maintain both local (no docker) and docker dev environments, we should have links to "Getting started with Docker" and "Getting started locally without Docker (Windows only)". The latter would point to the existing docs which is hard coded for Windows development until if/when we create directions for setting up dev environments on MacOS, Linux, etc.

Reply:
I concur with Keith on that..If the readme is simple enough, it should go in the readme, however more complex instructions should be outside of it.

Reply:
I've simplified the 'Getting Started' section with pointers to Docker and Windows Local instructions.